src/pe/header.rs

```diff
@@ -775,11 +795,12 @@ impl CoffHeader {
 ///
 /// The PE header is located at the very beginning of the file and
 /// is followed by the section table and sections.
-#[derive(Debug, PartialEq, Copy, Clone, Default)]
-pub struct Header {
+#[derive(Debug, PartialEq, Clone, Default)]
+pub struct Header<'a> {
     pub dos_header: DosHeader,
     /// DOS program for legacy loaders
-    pub dos_stub: DosStub,
```

Comment:
breaking change, and unclear why it went from `DosStub` to `&[u8]`

Reply:
Solved in b1b841cea67fd4fbd7d0ae948dafb6b9d66a346f.
How's it now? if it looks good for you I'd like you to make this change request solved on your end :)

Reply:
> i was surprised to notice the latter is a longer slice, e.g. comparing the two:

So-called rich header is within DOS stub bytes. So the `stub2` contains whole data of rich header in its tail. :)